lib/chef/provisioning/aws_driver/aws_resource.rb

```diff
@@ -136,7 +136,14 @@ def self.get_aws_object_id(value, **options)
 
   protected
 
-  NOT_PASSED = Object.new
+  # This supports the introduction of the NOT_PASSED chef constant
```

Comment:
This will work, but recall that in cheffish we fixed this with this instead:

1. Open up the `Chef` class directly so you get `NOT_PASSED` (instead of doing `module Chef::...` at the top, do `class Chef` and then `module ...`)

2. Define it like this:

```ruby
NOT_PASSED = Object.new if !defined?(NOT_PASSED)
```